# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1523298658 25200
#      Mon Apr 09 11:30:58 2018 -0700
# Branch stable
# Node ID e3d5c1def2c8690972beb7411357b56c4a7793ed
# Parent  f04a5543afda2c2a345726a4ba3c2baa3cfab9fd
doc: fix remaining mention of old name for Octave:data-file-in-path (bug #53588)

* warning_ids.m: Change remaining mention of Octave:fopen-file-in-path.  Fix
small typo in doc string.

diff --git a/scripts/help/warning_ids.m b/scripts/help/warning_ids.m
--- a/scripts/help/warning_ids.m
+++ b/scripts/help/warning_ids.m
@@ -177,18 +177,18 @@
 ## By default, the @code{Octave:eigs:UnconvergedEigenvalues} warning is
 ## enabled.
 ##
 ## @item Octave:data-file-in-path
 ## If the @code{Octave:data-file-in-path} warning is enabled, a warning is
 ## issued when Octave does not find the target of a file operation such as
 ## @code{load} or @code{fopen} directly, but is able to locate the file in
 ## Octave's search @code{path} for files.  The warning could indicate that a
-## different file target then the programmer intended is being used.
-## By default, the @code{Octave:fopen-file-in-path} warning is enabled.
+## different file target than the programmer intended is being used.
+## By default, the @code{Octave:data-file-in-path} warning is enabled.
 ##
 ## @item Octave:function-name-clash
 ## If the @code{Octave:function-name-clash} warning is enabled, a
 ## warning is issued when Octave finds that the name of a function
 ## defined in a function file differs from the name of the file.  (If
 ## the names disagree, the name declared inside the file is ignored.)
 ## By default, the @code{Octave:function-name-clash} warning is enabled.
 ##
